src/newLaunch/baseStage.scss

```diff
@@ -37,9 +37,11 @@
   .content-block {
```

Comment:
Can you please revert all unrelated formatting changes?

Reply:
done